server/src/main/java/org/apache/kafka/server/metrics/ClientMetricsConfigs.java

```diff
@@ -14,7 +14,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package kafka.metrics;
+package org.apache.kafka.server.metrics;
```

Comment:
@junrao @apoorvmittal10 Note that I have moved these recently introduced classes to this new module.

Reply:
Thanks @ijuma. @junrao I ll move the classes in other PRs to server/src package itself once review is complete. Moving prior active review will lose context of comments.

Reply:
Can you please share the PRs where this is being discussed now?